### DISCUSSION TITLE
Feature request: Add ability to configure language and run as startup options

### DISCUSSION THREAD

**Discussion by hamerstandr** [File-New-Project/EarTrumpet#675] [Ideas]
![image](https://user-images.githubusercontent.com/16276736/50050807-cfe24200-011a-11e9-9cf6-6c56bc7023b4.png)

**Comment by riverar** [comment]
What's the issue?

**Comment by hamerstandr** [comment]
> What's the issue?

1-set To Startup in Setting
2- change Language in Setting
